File: c course/c w3resouse questons/string/q6.c
#include<stdio.h>
int main()
{
    char x[50],y[50],m=0,n=0;
    int i=0;
    printf("enter the string x:");
    gets(x);
    printf("enter the string y:");
    gets(y);
    while(x[i]!='\0'||y[i]!='\0')
    {
        if(x[i]==y[i])
        {
            m++;
        }
        else
        {
            n++;
        }
        i++;
    }
    if(n>0)
    {
        printf("string are not equal");
    }
    else{
        printf("string are equal");
    }
    return 0;
}

File: Python/string/q6.py
String1 = input('Enter the String :')
#initialize empty String
String2 = ''
for i in String1:
    #check for alphabets
    if i not in ((ord(i) >= 65 and ord(i) <= 90)\
         or (ord(i) >= 97 and ord(i) <= 122)):
        #concatenate to empty string
        String2+=i
print('Alphabets in string are :' + String2)